src/main/resources/site/lib/site.js

```diff
@@ -3,32 +3,21 @@ var libs = {
 	util: require('/lib/enonic/util/util')
 };
 
-// "Globals":
 var appNamePropertyName = app.name.replace(/\./g,'-');
 
 function getConfig() {
 	return libs.portal.getSiteConfig();
 }
-function commaStringToArray(string) {
-	var arr = [];
-	string = string.replace(/ /g,''); // Remove all spaces in the string
-	arr = string.split(',');
-	return arr;
+function commaStringToArray(str) {
+	return str.split(',').trim();
 }
-function findValueInJson(json,paths) {
-	var value = null;
-	var pathLength = paths.length;
-	var jsonPath = ";"
-	for (var i = 0; i < pathLength; i++) {
-		if ( paths[i] ) {
-			jsonPath = 'json.data["' + paths[i] + '"]'; // Wrap property so we can have dashes in it
-			if ( eval(jsonPath) ) {
-				value = eval(jsonPath);
-				break; // Expect the first property in the string is the most important one to use
-			}
-		}
-	}
-	return value;
+function findValueInJson(json, paths) {
+	var hits = paths.filter(function(path) { return json.data[path]; });
+	return hits.length ? hits[0] : false;
+}
```

Comment:
This new code is short and sweet. But it doesn't return to me the actual value stored at that json location. When I add the eval() for this it has trouble with empty values and paths that does not exists. Previous code checked for the existance of the path before trying to fetch stored data. So I'll think I'll go ahead and not merge your changes here and just revert to how it was.